src/base-element.js

```diff
@@ -398,6 +398,18 @@ export class BaseElement {
     return false;
   }
 
+  /**
+   * Subclasses can override this method to indicate that it is has
+   * render-blocking service.
+   *
+   * The return value of this function is used to determine if the element
+   * built _and_ laid out will be prioritized.
+   * @return {boolean}
+   */
+  isRenderBlocking() {
```

Comment:
I'd rather this be specific to `buildCallback`, with "build" somewhere in the name. Having this return true currently doesn't do anything for `layoutCallback`.

Reply:
to confirm, you want me to change the name? 
What about `isBuildRenderBlocking()`?

Reply:
Yah, that's good.

Reply:
SG. Renamed.